app/src/main/java/com/nmakademija/nmaakademija/entity/Article.java

```diff
@@ -1,8 +1,9 @@
 package com.nmakademija.nmaakademija.entity;
 
-import java.io.Serializable;
+import android.os.Parcel;
+import android.os.Parcelable;
 
-public class Article implements Serializable {
+public class Article implements Parcelable {
```

Comment:
We use `Parcelable` here instead of `Serializable`. It's far faster than `Serializable`. See http://www.developerphil.com/parcelable-vs-serializable/